packages/renderer/src/lib/configmaps-secrets/ConfigMapSecretList.svelte

```diff
@@ -151,6 +153,7 @@ const row = new TableRow<ConfigMapSecretUI>({ selectable: _configmapSecret => tr
 <NavPage bind:searchTerm={searchTerm} title="configmaps & secrets">
   <svelte:fragment slot="additional-actions">
     <KubeApplyYamlButton />
+    <Button on:click={handleNavigation.bind(undefined, { page: NavigationPage.KUBERNETES_RESOURCE_CREATE })}>Create Resource</Button>
```

Comment:
issue: blocking: can we have a pre-req PR that would refactor the code and introduce a component like <KubeAdditionalActions> or introduce a KubernetesNavPage with the additional actions included there or any solution like that to avoid to copy/paste the exact same lines across 6-7 pages when we need to add a button in all the kube pages ?

Reply:
Yeah, sure, will do that!